kernel/thread/thread.c

```diff
@@ -122,22 +123,35 @@ int thd_each(int (*cb)(kthread_t *thd, void *user_data), void *data) {
 int thd_pslist(int (*pf)(const char *fmt, ...)) {
     kthread_t *cur;
 
+    const uint64_t ns_time = perf_cntr_timer_ns();
```

Comment:
Unless I missed something this should probably move to inside the `if(perf_cntr_timer_enabled())` check right? Because the value is unused if we don't have them enabled, so we'd be polling the fallback regular timer for no reason.

Reply:
Ah ok, that makes perfect sense.  I figured I was just missing why but there was likely a good one :)